internal/pkg/template/templates/environment/partials/s3-elb-access-logs.yml

```diff
@@ -0,0 +1,39 @@
+ELBAccessLogsBucketPolicy:
+  Type: AWS::S3::BucketPolicy
+  Condition: CreateALB
+  DependsOn: ELBAccessLogsBucket
+  Properties:
+    Bucket: !Ref ELBAccessLogsBucket
+    PolicyDocument:
+      Version: '2012-10-17'
+      Statement:
+        - Action:
+            - s3:PutObject
+          Effect: Allow
+          Resource:
+            - !Join
+              - ''
+              - - 'arn:'
+                - !Ref AWS::Partition
+                - ':s3:::'
+                - !Ref ELBAccessLogsBucket
+                - '/'
+                - {{ .BucketPrefix }}
+                - '/AWSLogs/'
+                - !Ref AWS::AccountId
+                - '/*'
+          Principal:
+            AWS: !Join [ "", [  "arn:aws:iam::", !FindInMap [ RegionalConfigs, !Ref 'AWS::Region', ELBAccountId ], ":root" ] ]
```

Comment:
Can we replace "arn:aws:iam" with `!Sub "arn:${AWS::Partition}:iam` (I don't remember the exact syntax) so that it uses the appropriate partition for the region